src/Service/PermissionService.php

```
@@ -0,0 +1,337 @@
+    /**
+     * Gets all permissions a user has on an organization.
+     *
+     * @return list<string>
+     */
+    public function getUserOrganizationPermissions(User $user, Organization $organization): array
+    {
+        $permissions = [];
+
+        // Check personal organization ownership
+        if ($this->isPersonalOrganizationOwner($user, $organization)) {
+            return Permission::getAllOrganizationPermissions();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Security\Permission' in method 'getUserOrganizationPermissions'. (undefined)

(StaticAccess)